flow-typed/custom/@google-cloud/trace-agent.js

```diff
@@ -37,12 +37,15 @@ declare function traceagent$start(
     config?: traceagent$Config,
 ): traceagent$Tracer;
 
+declare function traceagent$get(): traceagent$Tracer;
+
 declare module "@google-cloud/trace-agent" {
     declare export type Config = traceagent$Config;
     declare export type SpanOptions = traceagent$SpanOptions;
     declare export type Span = traceagent$Span;
     declare export type Tracer = traceagent$Tracer;
     declare module.exports: {
         start: typeof traceagent$start,
+        get: typeof traceagent$get,
```

Comment:
This is part of the API, I just hadn't bothered adding it to the flow types before because we weren't using it.